app/helpers/browse_helper.rb

```diff
@@ -3,8 +3,44 @@ def display_popular_links_for_slug?(slug)
     I18n.exists?(slug.to_s, scope: "browse.popular_links")
   end
 
+  def variant_a_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_b_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_c_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_z_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def no_popular_tasks_ab_test?
+    !respond_to?(:popular_tasks_variant)
+  end
+
+  def variant_popular_links_for_slug(slug)
+    default_variant_popular_links = variant_c_popular_links(slug)
+
+    if no_popular_tasks_ab_test?
```

Comment:
I'm not sure I follow the branching here, I think we need
```
if popular_tasks_variant_a_page?
# show variant a links
elsif popular_tasks_variant_b_page?
# show variant b links
elsif popular_tasks_variant_c_page?
# show original  (links||nothing)
else (variant z)
# show original (links||nothing)
end
```

The `links||nothing` would come from `default_variant_popular_links`

Reply:
Done.